seller-info-service/src/test/resources/application-testcontainers.yml

```
@@ -1,4 +1,7 @@
 spring:
+  app:
+    seller:
+      url: /public/api/v1/sellers
```

```suggestion
      url: sellers
```
У нас есть пакет `com.yandex.market.util`
Где содержится `public/api/v1`
Поэтому лучше поменяй в этом пре `@RequestMapping`, чтобы потом это не делать